specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ManagedLedgerDigestUploads.json

```diff
@@ -0,0 +1,335 @@
+          },
+          "202": {
+            "description": "Updating the ledger digest upload configuration is in progress."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Enables managed ledger digest upload configuration for a database": {
+            "$ref": "./examples/ManagedLedgerDigestUploadsEnable.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/databases/{databaseName}/ledgerDigestUploads/{ledgerDigestUploads}/disable": {
+      "post": {
+        "tags": [
+          "ManagedLedgerDigestUploads"
+        ],
+        "description": "Disables uploading ledger digests to an Azure Storage account or an Azure Confidential Ledger instance.",
+        "operationId": "ManagedLedgerDigestUploads_Disable",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "ledgerDigestUploads",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "current"
+            ],
+            "x-ms-enum": {
+              "name": "ManagedLedgerDigestUploadsName",
+              "modelAsString": true
+            }
+          },
```

Comment:
It is good to define this parameter below in common, as just refer it everywhere to avoid typo and future multiple edits overhead to this parameter.

Reply:
we will move it to common 2023. 
actually, the swagger /example files from SQL are auto-generated from our upstream SQL repo. All these defs are just in one place in our codebase. It is duplicated in multiple places by our swagger auto generator.